### PR TITLE
First draft for restructuring rpc and adding grest schema to dbsync

### PR DESCRIPTION
**Pull request by Scitz0**
CNTools not updated yet to reflect the changes.

### CODE REVIEW

**Review by rdlrt — APPROVED** [review]
While we need to test overall updates, this PR with it's scope LGTM, wont be able to test until monday tho

We might need to add a changelog entry

File: files/grest/rpc/pool/pool_update.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+CREATE OR REPLACE FUNCTION grest.pool_update(_pool_bech32 text, _current_epoch_no numeric default 0, _state text default '' )
```
Comment (rdlrt) [inline files/grest/rpc/pool/pool_update.sql:1]: We could potentially call this one pool_updates (plural)?

Comment (Scitz0) [reply files/grest/rpc/pool/pool_update.sql:1]: 👍

**Review by DamjanOstrelic — APPROVED** [review]
LGTM 👍 Will be able to test towards the end of the day today if it will still be needed.

**Pull request merged**